bin/desi_tiles_completeness

```diff
@@ -22,7 +22,7 @@ parser.add_argument('-i','--infile', type=str, default=None, required=False,
                     help = 'Input exposure summary file')
 parser.add_argument('-o','--outfile', type=str, default=None, required=True,
                     help = 'Output tiles completeness file')
-parser.add_argument('--prod', type = str, default = None, required=False,
+parser.add_argument('--prod', type = str, default = "daily", required=False,
                     help = 'Path to input reduction, e.g. /global/cfs/cdirs/desi/spectro/redux/blanc/,  or simply prod version, like blanc, but requires env. variable DESI_SPECTRO_REDUX. Default is $DESI_SPECTRO_REDUX/$SPECPROD.')
```

Comment:
Help string no longer matches the new default, and I think I like what the previous default was (using $SPECPROD instead of defaulting to "daily").  I'm imagining a future error where we think we are calculating tile completeness for a production and it is accidentally defaulting to calculating it off of daily but writing it into the new production....